astropy/visualization/wcsaxes/core.py

```diff
@@ -96,6 +96,8 @@ class WCSAxes(Axes):
     def __init__(self, fig, rect, wcs=None, transform=None, coord_meta=None,
                  transData=None, slices=None, frame_class=None,
                  **kwargs):
+        """
+        """
```

Comment:
@bsipocz @pllim this is what was causing the issue with Sphinx 2.2, the ``__init__`` docstring was being automatically inherited. Not a bug in Sphinx (in fact it was a bug that it wasn't inherited before)